roles/content_view_promote/README.md

```diff
@@ -0,0 +1,127 @@
+theforeman.foreman.content_view_promote
+=========
+
+A role for automating the staging/promotion of Foreman Content-Views through various Organizations and LifeCycle Environemnts.
+
+Requirements
+------------
+
+This role requires the theforeman.foreman module collection.
+
+Role Variables
+--------------
+
+The primary dictionary is the organizations dictionary, which is formatted as such:
+```
+organizations:
```

Comment:
I think this tends to break our general model of crafting roles that follow along tenet-lines. Is it common in your experience to be performing publish and promotes in different organizations from the same workflow?

@evgeni @mdellweg @wbclark your thoughts welcome

Reply:
@ehelms another thought here:

If there are opinions or tenets that the foreman team wants contributors to follow, there needs to be some documentation stating such.  coming in after a commit and saying something isn't following an undocumented guideline makes for a pretty frustrating contributor experience

Reply:
we're working on this in https://github.com/theforeman/foreman-ansible-modules/pull/1186 :)